### ISSUE TITLE
stratum ports 

### ISSUE THREAD

**Issue by mario1987** [bonesoul/CoiniumServ#818]
I am trying to use more ports with different diff's  is this application able to do that ??

"ports": {
	 "3032": { }
}

this does not seem to work

**Comment by bonesoul** [comment]
nope not yet.

**Comment by jdghenderson** [comment]
Yes this would be great as then we could offer Vardiff and some fixed difficulty ports as well as some other pool implementations have.

**Comment by bonesoul** [comment]
marking this as help-needed then as it'll be a great start for new comers :)
